CHANGES.rst

```diff
@@ -166,6 +166,9 @@ astropy.units
   ``bool(0 * u.dimensionless_unscaled)`` evaluates to ``True``. In the future,
   attempting to convert a ``Quantity`` to a ``bool`` will raise ``ValueError``.
   [#6580, #6590]
+ - Modify the ``brightness_temperature`` equivalency to provide a surface
```

Comment:
Yikes, bad rebase.

Reply:
...that's weird, there were no merge conflicts.   I don't see any problem here... unless you're referring to the space at the beginning of the line?  That didn't come from the rebase.